frontend/src/components/Songs/SongInfo.js

```diff
@@ -67,17 +64,13 @@ const SongInfo = () => {
     } catch (err) {}
   };
 
-  function handlePlayPause(song) {
-    const audioElement = document.getElementById("audio");
-    if (!isPlaying) {
-      audioElement?.play(song?.url);
-      setIsPlaying(true);
-      player(song?.url);
-      // console.log("audioEle", audioElement.play());
+  const handlePlayPause = () => {
+    if (song.id !== playerSong) {
+      dispatch(getAudio(audioFile));
+    } else if (isPlaying) {
+      dispatch(pauseAudio());
     } else {
-      audioElement?.pause(song?.url);
-      // console.log("pause");
-      setIsPlaying(false);
+      dispatch(playAudio());
     }
```

Comment:
Now that the updating the audio store status will also update the audio player, dispatching `pauseAudio` and `playAudio` actions should work from anywhere.